lib/shelley/src/Cardano/Wallet/Shelley/Pools.hs

```diff
@@ -682,24 +707,45 @@ monitorStakePools tr gp nl DBLayer{..} =
 
 -- | Worker thread that monitors pool metadata and syncs it to the database.
 monitorMetadata
-    :: Tracer IO StakePoolLog
+    :: Tracer IO (WorkerLog Text StakePoolLog)
+    -> Tracer IO StakePoolLog
```

Comment:
This seems wrong. Having two tracers is a good sign that something is wrong here. The monitorMetadata should not be aware of its parent tracer.